### PR TITLE
Updated to latest Vinery

### PR DESCRIPTION
**Pull request by Groupix05**
Requires 1.3.6-BETA01 dynamic trees

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -13,12 +13,12 @@ loader_version_range=[47,)
...
 dynamictrees_version_range=[0,)
 target_mod_id=vinery
-target_mod_version_range=[1.4.25,)
+target_mod_version_range=[1.4.32,)
 
 mod_id=dtvinery
 mod_name=Dynamic Trees for Vinery
 mod_name_s=DynamicTreesforVinery
-mod_version=1.1
+mod_version=1.2
```
Comment (joe-vettek) [inline gradle.properties:21]: A change in the second version number would require a major change in the mod content, which currently does not exist.

**Review by joe-vettek — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (Groupix05) [reply gradle.properties:21]: I changed the version to 1.2 because it's only compatible with latest Vinery. Do you want me to change it to 1.1.1 ?

**Comment by joe-vettek** [comment]
I haven't figured out how to modify this PR yet, so I pushed 1.1.1 to fix this minor issue. I've been a bit busy recently and can't study it carefully sorry

**Comment by joe-vettek** [comment]
Thanks for reminding me that Vinery has updated.

**Pull request closed without merging**